src/core/src/bootstrap/Constants.py

```diff
@@ -183,6 +183,7 @@ class StatusTruncationConfig(EnumBackport):
         MIN_ASSESSMENT_PATCHES_TO_RETAIN = 5
         TRUNCATION_WARNING_MESSAGE = "Package lists were truncated to limit reporting data volume. In-VM logs contain complete lists."
         TURN_ON_TRUNCATION = True
+        NO_TRUNCATION_LOGIC_IN_SEC = 60
```

Comment:
rename to TIME_BETWEEN_TRUNCATION_IN_SEC or DELAY_TRUNCATION_BY_SEC

Reply:
hmm i changed the constant to that, but here it is showing the outdated var name:

![image](https://github.com/Azure/LinuxPatchExtension/assets/127874208/1b027df1-0473-42c3-8f1e-8ce19b62a519)

Reply:
DELAY_TRUNCATION_BY_SEC is the updated name correct? what is the outdated var name then?

Reply:
the outdated var name was NO_TRUNCATION_LOGIC_IN_SEC